src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -114,4 +131,30 @@ void notRemoveNonOrphans() {
         // then
         assertThat(authorService.count()).isOne();
     }
+    
+    @Test
+    void notEmptyFirstName() {
+    	// given
+    	Author noFirst = new Author(" ", "Gaarder");
+    	
+    	validator = Validation.buildDefaultValidatorFactory().getValidator();
+    	Set<ConstraintViolation<Author>> violations = validator.validateProperty(noFirst, "firstName");
+    	
+    	//  then
+    	assertThat(violations.size()).isEqualTo(1);
+    }
+    
+    @Test
+    void notEmptyLastName() {
```

Comment:
![image](https://user-images.githubusercontent.com/11173328/98268899-fadf1480-1f84-11eb-9325-37117ffae072.png)

Every test class has the name of the class it's testing followed by 'should'. In `AuthorTest`, you'll see the display name is 'Author should'. Each test method follows on from that to complete the sentence (e.g. `Author should` ... `notRemoveNonOrphans`).

Could you please rename your methods to follow this style? For example, this method could be renamed to something like 'notAcceptEmptyLastName' (so that we get `Author should notAcceptEmptyLastName`)